samples/apps/bookinfo/src/ratings/ratings.js

```diff
@@ -17,9 +17,20 @@ var dispatcher = require('httpdispatcher')
 
 var port = parseInt(process.argv[2])
 
+/**
+ * We default to using mongodb, if DB_TYPE is not set to mysql.
+ */
 if (process.env.SERVICE_VERSION === 'v2') {
-  var MongoClient = require('mongodb').MongoClient
-  var url = process.env.MONGO_DB_URL
+  if (process.env.DB_TYPE === 'mysql') {
```

Comment:
thanks!
is this mostly reverting to what it was before ?

Reply:
yes. Reverting and cleaning up. We can now use API driven tests and no longer need html comparators

Reply:
@ldemailly In this implementation, the `DB_TYPE` variable needs to be set to `mysql` for MySQL to be used. This is in line with changes @rshriram did here: https://github.com/istio/istio/commit/22014c8a0a71b465cdc900dd6a364888003c7e5d#diff-6a25b9bedba073dde47bfc3559d560a1